### DISCUSSION TITLE
AuthRetryableFetchError: Load failed

### DISCUSSION THREAD

**Discussion by Kaseax** [supabase/supabase#18230] [Questions] [answered]
Hey, just trying to get supabase first time into my NextJS Application. I'm getting this error at registration:

`AuthRetryableFetchError: Load failed (anonyme Funktion) — client.js:1:910 (anonyme Funktion) — setup-hydration-warning.js:18 (anonyme Funktion) — signup.tsx:18 asyncFunctionResume (anonyme Funktion) promiseReactionJobWithoutPromise promiseReactionJob signup.tsx:21`

that's the code: 

```
    async function signUpWithEmail() {
        try {
            if (email && password) {
                console.log('Email: ', email);
                console.log('Password: ', password);
                const response = await supabase.auth.signUp({
                    email: email,
                    password: password,
                });
                if (response.error) {
                    console.error('Autherror: ', response.error);
                }
                const userId = response.data.user?.id;
                console.log(userId);
            }
        } catch (error) {
            console.error('Fehler bei der Registrierung:', error);
        }
    }
```

**Comment by kevcodez** [comment]
Could you show how you set up the Supabase client and also post the Supabase URL you've used? Could just be a wrong Supabase URL.

Also, have you ensured your project is not paused?

**Reply by Kaseax** [reply]
that's the supabaseClient.ts file:
```
import { createClient } from "@supabase/supabase-js";

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL ?? "";
const supabaseAnonKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY ?? "";

const supabase = createClient(supabaseUrl, supabaseAnonKey);

export default supabase;
```

that's the supabaseURL: https://wszgcbazieqplpgzxbrp.supabase.co

yes, my project is active.

**Reply by kevcodez** [reply]
I don't think we prepend https or anything, so the supabaseURL would have to be `https://wszgcbazieqplpgzxbrp.supabase.co`

**Reply by Kaseax** [reply]
isn't that exactly what i have? that's what it says in the .env.local, as i said.

**Reply by kevcodez** [reply]
Ah sorry, GitHub shortened the URL and removed the protocol at the beginning - missed it. Are you seeing failed network calls in your browser, could you post them here?

**Reply by Kaseax** [reply]
I See the network call, yes. but there's no response or similar, I could share. If I copy  the HTTP-Response, there's just "NaN null"

**Reply by Kaseax** [reply]
That's the request:
POST /auth/v1/signup
Content-Type: application/json;charset=UTF-8
Authorization: Bearer ey...
Origin: http://localhost:3000
Referer: http://localhost:3000/
Accept: */*
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.1 Safari/605.1.15
apikey: ey...
X-Client-Info: supabase-js-web/2.38.1

**Reply by kevcodez** [reply]
It would be great to share the entire request/response, i.e. by exporting a HAR file through your browser

Are you sure the env var for the supabase url is set correctly? Can you log it before initializing the supabase client to ensure it's correctly loaded from your env?

**Reply by Kaseax** [reply]
GitHub wont support HAR files, here's the uploaded file: http://cdn.kaseax.de/files/localhost.har

I got "https://wszgcbazieqplpgzxbrp.supabase.co" in my console, so yes.

**Reply by kevcodez** [reply]
Don't see a call to `/auth/v1/signup` in the HAR file - is the signup called from the server-side with Next? Are you using the [auth-helpers](https://supabase.com/docs/guides/auth/auth-helpers/nextjs)?

Could you please provide a minimal reproducible repository for us to take a closer look? Otherwise, it's very difficult to debug without the full context

**Reply by Kaseax** [reply]
just made the repository public, so you can see it, until the problem is gone: https://github.com/Kaseax/LinkSpotlight

Safari tells me in the network tab, that the request Is made on following url: https://wszgcbazieqplpgzxbrp.supabase.co/auth/v1/signup

**Reply by Kaseax** [reply]
hey! is there anything new?

**Reply by kevcodez** [reply, marked as answer]
Didn't have any time to actually check out your repository yet, but it looks like you manually wired up Supabase. You could try following our official guides/templates and use the auth helpers - 

https://supabase.com/docs/guides/getting-started/quickstarts/nextjs
https://supabase.com/docs/guides/auth/auth-helpers/nextjs

**Reply by Kaseax** [reply]
Tried it again, that's the new response:
`[Error] TypeError: Load failed
	(anonyme Funktion) (client.js:2:918)
	(anonyme Funktion) (setup-hydration-warning.js:18)
	(anonyme Funktion) (instrument.js:117)
	(anonyme Funktion) (fetch.js:74)
	asyncFunctionResume
	(anonyme Funktion)
	promiseReactionJobWithoutPromise
	promiseReactionJob`

**Reply by Kaseax** [reply]
ah, didn't see your response. if there's no other solution, I will do that, yes.

**Reply by Kaseax** [reply]
with the template, it worked now. thanks!
